core/math/math_2d.h

```diff
@@ -107,9 +107,9 @@ struct Vector2 {
 	Vector2 cubic_interpolate(const Vector2 &p_b, const Vector2 &p_pre_a, const Vector2 &p_post_b, real_t p_t) const;
 	Vector2 cubic_interpolate_soft(const Vector2 &p_b, const Vector2 &p_pre_a, const Vector2 &p_post_b, real_t p_t) const;
 
-	Vector2 slide(const Vector2 &p_vec) const;
-	Vector2 bounce(const Vector2 &p_vec) const;
-	Vector2 reflect(const Vector2 &p_vec) const;
+	Vector2 slide(const Vector2 &p_n) const;
+	Vector2 bounce(const Vector2 &p_n) const;
+	Vector2 reflect(const Vector2 &p_n) const;
```

Comment:
I was going to change it to p_normal, but I was not sure if I should, since the issue was making the parameter names consistent, not necessarily changing them.

I'll change this to p_normal :+1: